bugbug/db.py

```diff
@@ -152,6 +152,14 @@ def read(self):
             pass
 
 
+class LastModifiedNotAvailable(Exception):
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        return self.message
```

Comment:
We can implement this as an empty class so it inherits everything from its parent `Exception`:
```
class LastModifiedNotAvailable(Exception):
    pass
```

Reply:
implemented